hack/verify-structured-logging.sh

```diff
@@ -48,15 +48,28 @@ while IFS='' read -r line; do
   migrated_packages+=("$KUBE_ROOT/$line")
 done < <(cat "${migrated_packages_file}")
 
-
+echo "Running structured logging static check on migrated packages"
 ret=0
 GOOS=linux    logcheck "${migrated_packages[@]}" || ret=$?
 GOOS=windows  logcheck "${migrated_packages[@]}" || ret=$?
 
 if [ $ret -eq 0 ]; then
-  echo "Structured logging static check is passed :)."
+  echo "Structured logging static check passed on migrated packages :)"
+else
+  echo "Please fix above failures. You can locally test via:"
+  echo "hack/verify-structured-logging.sh"
+fi
+
+echo -e "\nRunning structured logging static check on all packages"
+ret=0
+GOOS=linux    logcheck -allow-unstructured `go list ${KUBE_ROOT}/...` || ret=$?
```

Comment:
If there is any failure migrated package it will be reported twice, fist time by checking migrated packages and second time when checking whole repo. This will be confusing to read by user. Could you rewrite script so that it will report failure only once?

Reply:
Yes, I'm looking into it.
Looks like we also need to skip checking some packages that are generated and test only etc. This is going to add to overall test time.

Reply:
K8s verify commands can take more than 2h, we don't care about increasing runtime by 10s. Let's focus first on user experience and correctness, and later on performance.

Reply:
Was able to list all packages and filter migrated packages using `go list` and grep only.
Checks now run once on migrated packages and once on all other packages.